lib/phx_tools_web/live/phx_tools_components.ex

```diff
@@ -0,0 +1,268 @@
+                </div>
+              </div>
+            </div>
+            <div class="w-full bg-blue-600 flex justify-end">
+              <.os_instructions live_action={@live_action} operating_system={@operating_system} />
+            </div>
+            <div class="justify-center gap-4 md:flex">
+              <div class="sm:grid grid-cols-2 gap-4 sm:py-4">
+                <.command_select_button
+                  live_action={@live_action}
+                  operating_system={@operating_system}
+                />
+                <.source_code_button source_code_url={@source_code_url} />
+              </div>
+            </div>
+          </div>
+          <div class="lg:w-[1100px]">
+            <div>
+              <p class="text-xs px-2 leading-5 text-center text-white font-martian md:mb-5 md:text-sm md:leading-6 lg:text-base">
+                Phx.tools is a shell script for Linux and macOS that configures the development environment for you in a few easy steps. Once you finish running the script, you'll be able to start the database server, create a new Phoenix application, and launch the server.
+              </p>
+            </div>
+            <div
+              class="mx-auto"
+              data-asciicast={
+                if @live_action == :macOS,
+                  do: "bJMOlPe5F4mFLY0Rl6fiJSOp3",
+                  else: "XhDpRstBJ4df2gfiRfp0awDPO"
+              }
+              id={"asciinema-#{Utils.random_id()}"}
+              phx-hook="AsciinemaHook"
+            >
+            </div>
+
+            <p class="md:my-5 text-white md:px-12 text-xs md:text-sm lg:text-base font-martian">
+              Read about website updates here -
+              <a
+                href="https://optimum.ba/blog/exciting-updates-to-phx-tools"
+                class="text-[#24B2FF] font-martian text-sm underline"
+                target="_blank"
+              >
+                https://optimum.ba/blog/exciting-updates-to-phx-tools
+              </a>
+            </p>
+          </div>
+        </div>
+      </div>
+      <.footer />
+    </div>
+    """
+  end
+
+  @spec os_instructions(assigns()) :: rendered()
+  def os_instructions(assigns) do
+    ~H"""
+    <div
+      id="installation-instructions"
+      phx-click-away={JS.hide(to: "#installation-instructions")}
+      class="hidden font-normal solved-height font-martian bg-[#110A33] absolute md:w-[90%]"
+    >
+      <div class="block">
+        <div class="h-full shadow-[#C2B8FF] shadow-md rounded-md pb-2">
+          <div class="text-start px-[3%] lg:text-xl md:text-lg sm:text-md">
+            <h1 class="text-white text-center text-sm md:text-base lg:text-xl lg:my-[5%] md:my-[2%] sm:my-[2%] lg:pt-5">
+              <%= @live_action %> installation process
+            </h1>
+            <ol class="list-decimal ml-3 pl-5 text-xs md:text-sm lg:text-base text-white lg:mt-4 sm:mt-2 leading-6">
+              <%= for instruction <- render_instructions(@live_action) do %>
```

Comment:
I feel like this could be better with slots.
https://hexdocs.pm/phoenix_live_view/Phoenix.Component.html#module-slot-attributes